Products/zms/zpt/ZMS/manage_customize.zpt

```diff
@@ -335,10 +335,7 @@
 				<div class="form-group row">
 					<label class="col-sm-2 control-label">ZMS Version</label>
 					<div class="col-sm-10">
-						<a title="ZMS commits on github.com" target="_blank"

```

Comment:
Move `<a href>` to zms.zms_version() -> `rtn += ' git#%s'%(FETCH_HEAD)`

Reply:
DONE by https://github.com/zms-publishing/ZMS/commit/a637c49799f228daf973f8860473c67337dfd280